packages/graphback-cli/src/components/db.ts

```diff
@@ -13,57 +12,49 @@ const handleError = (err: { code: string; message: string; }): void => {
     logError(err.message)
   }
   process.exit(0)
-}
+};
 
 export const createDBResources = async (cliFlags: { project?: string }): Promise<any> => {
   let databaseOperations: any;
   try {
     const config = await loadConfig({
       rootDir: process.cwd(),
-      extensions: [graphbackConfigExtension]
+      extensions: [dbMigrationConfigExtension]
     });
-    const project = config.getProject(cliFlags.project || 'default')
-    const graphbackConfig = project.extension(extensionName);
+    const project = config.getProject(cliFlags.project || 'default');
+    const dbMigrationConfig = project.extension(dbmigrationsExtension);
 
-    if (!graphbackConfig) {
-      throw new Error(`You should provide a valid '${extensionName}' config to generate schema from data model`);
+    if (!dbMigrationConfig) {
+      throw new Error(`You should provide a valid '${dbmigrationsExtension}' config to migrate schema`);
     }
 
-    if (!graphbackConfig.model) {
-      throw new Error(`'${extensionName}' config missing 'model' value that is required`);
-    }
-
-    const models = new GlobSync(`${graphbackConfig.model}/*.graphql`)
+    const models = new GlobSync(project.schema as string);
```

Comment:
You could define `const schema = await project.getSchema("string");` before this and use it here.